.aicoe-ci.yaml

```diff
@@ -1,15 +1,37 @@
 check:
   - thoth-build
   - thoth-precommit
-build:
-  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
-  build-stratergy: Source
-  registry: quay.io
-  registry-org: thoth-station
-  registry-project: elyra-aidevsecops-tutorial
-  registry-secret: thoth-station-thoth-pusher-secret
-deploy:
-  project-org: "thoth-station"
-  project-name: "elyra-aidevsecops-tutorial"
-  image-name: "elyra-aidevsecops-tutorial"
-  overlay-contextpath: "manifests/overlays/test/imagestreamtag.yaml"
+  - overlays-build
+overlays_dir: overlays
+overlays:
+  - name: download-dataset
+    build:
+      base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+      build-stratergy: Source
+      registry: quay.io
+      registry-org: thoth-station
+      registry-project: elyra-aidevsecops-dataset
+      registry-secret: thoth-station-thoth-pusher-secret
+
+  - name: test-model
+    build:
+      base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+      build-stratergy: Source
+      registry: quay.io
+      registry-org: thoth-station
+      registry-project: elyra-aidevsecops-testing
+      registry-secret: thoth-station-thoth-pusher-secret
+
+  - name: inference
+    build:
+      base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+      build-stratergy: Source
+      registry: quay.io
+      registry-org: thoth-station
+      registry-project: elyra-aidevsecops-tutorial
+      registry-secret: thoth-station-thoth-pusher-secret
+    deploy:
```

Comment:
this is meant as a deploy per overlay?

Reply:
I think it deployed just the overlay you want to be deployed, in this case only the inference one. From Elyra point of view you don't need deployment of the other steps, as they are pulled from registry during Kubeflow Pipeline Run

Reply:
yes, we can have deploy the overlay to different places. 
So aicoe-ci supports the delivery per overlay.